README.md

```diff
@@ -131,3 +131,13 @@ The German government has asked SAP and Deutsche Telekom to develop the Corona-W
 [cwa-verification-portal]: https://github.com/corona-warn-app/cwa-verification-portal
 [cwa-verification-iam]: https://github.com/corona-warn-app/cwa-verification-iam
 [cwa-testresult-server]: https://github.com/corona-warn-app/cwa-testresult-server
+
+## Licensing
+
+Copyright (c) 2020 SAP SE or an SAP affiliate company.
+
+Licensed under the **Apache License, Version 2.0** (the "License"); you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0.
```

Comment:
@Ein-Tim: 
The link to the License should be changed and redirect to the license file within our repo:

```You may obtain a copy of the License from [here](./LICENSES/Apache-2.0.txt).```